### PR TITLE
Sign up as user with account

### PR DESCRIPTION
**Pull request by Alecodeco**
Basic authentication with Devise. Default auto-generated routes for model User.
Trello: https://trello.com/c/EQ1R87O0

### CODE REVIEW

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class PagesController < ApplicationController
```
Comment (santir489) [inline app/controllers/pages_controller.rb:1]: We can delete this controller, as this project is only API

File: app/helpers/pages_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module PagesHelper
```
Comment (santir489) [inline app/helpers/pages_helper.rb:1]: We can delete this helper, as it is empty

File: app/views/layouts/application.html.erb [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
```
Comment (santir489) [inline app/views/layouts/application.html.erb:2]: We can delete this, as this project is only API

File: app/views/pages/welcome.erb [hunk 4]
```diff
@@ -0,0 +1,4 @@
+<h1>Welcome</h1>
```
Comment (santir489) [inline app/views/pages/welcome.erb:1]: We can delete this, as this project is only API

File: test/controllers/pages_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
```
Comment (santir489) [inline test/controllers/pages_controller_test.rb:1]: We can delete this, as this project is only API

**Review by santir489 — COMMENTED** [review]
Nice. Left some comment

File: app/assets/stylesheets/pages.scss [hunk 6]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the pages controller here.
```
Comment (DJA89) [inline app/assets/stylesheets/pages.scss:1]: We can also delete the css if we won't be using it

File: Gemfile [hunk 7]
```diff
@@ -23,6 +23,7 @@ gem 'jbuilder', '~> 2.7'
 gem 'bcrypt', '~> 3.1.7'
 
 gem 'sqlite3'
+gem 'devise'
```
Comment (DJA89) [inline Gemfile:26]: Will you be adding `devise_token_auth` in a different PR?

Comment (Alecodeco) [reply Gemfile:26]: I will update this one! Thanks for the reminder, I almost forgot that one

**Pull request merged**